sdk/python/kfp/v2/components/experimental/component_spec.py

```diff
@@ -14,40 +14,36 @@
 """Definitions for component spec."""
 
 from dataclasses import dataclass
-from typing import Any, Mapping, Optional, Sequence, Union
+from typing import Any, Dict, Mapping, Optional, Sequence, Union
+from pydantic import BaseModel
```

Comment:
nit: import packages/modules but not classes/functions

https://google.github.io/styleguide/pyguide.html#22-imports